scripts/generate-git-snapshot

```diff
@@ -60,8 +60,14 @@ gbp_opts() {
   if [ -n "${GBP_OPTS:-}" ] ; then
     echo "Found environment variable GBP_OPTS, set to ${GBP_OPTS}"
   else
-    GBP_OPTS=" -nc --git-force-create --git-ignore-new --git-ignore-branch -S -us -uc --git-verbose --git-builder=/bin/true --git-cleaner=/bin/true --git-export-dir= "
     echo "Using git-buildpackage default options provided by jenkins-debian-glue"
+
+    if gbp config buildpackage.overlay | grep -q 'buildpackage.overlay=True' ; then
```

Comment:
This doesn't evaluate correctly with gbp (v0.9.14) on debian buster. `gbp config buildpackage.overlay` returns "True" not "buildpackage.overlay=True".

Suggest to change the grep to "True" or the command to `gbp config buildpackage`.